### PR TITLE
Apply Prettier everywhere

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

- Apply Prettier everywhere
- Check in Node version with `.nvmrc`
- Check Prettier in CI

## How to test

- Check actions
- Run `npm run lint` locally

## How can we measure success?

More consistency in formatting.

## Have we considered potential risks?

N/A

## Images

N/A

## Accessibility

N/A

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+18.18.2
```
Comment (georgeblahblah) [inline .nvmrc:1]: could be 20 now!

File: 6-generics/slides.md [hunk 2]
```diff
@@ -154,8 +154,8 @@ If you were writing the `Array` type yourself, it might look something like this
 
 ```ts
 type Array<Element> = {
-    length: number;
-}
+	length: number;
```
Comment (georgeblahblah) [inline 6-generics/slides.md:157]: is this a `tab`? One thing I noticed in the "rendered" slides is that tabs are wide, and so the code examples look quite spaced out. It also makes it more likely the line will cause horizontal scrolling.

Is there something we can do in the markdown=>html conversion to convert tabs to spaces?

**Review by georgeblahblah — APPROVED** [review]
Looks good! I had 2 comments:

* maybe we can go straight to Node 20 now
* tabs in the code examples don't display so well in the HTML render. Maybe we can convert tabs to spaces when we render.

File: 6-generics/slides.md (hunk 2 above)
Comment (mxdvl) [reply 6-generics/slides.md:157]: I think it’s best to just [set `tab-size` in CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/tab-size). Let me have a look.

Comment (mxdvl) [reply 6-generics/slides.md:157]: It works great!

image

**Pull request merged**